ci/official/utilities/setup.sh

```diff
@@ -61,13 +61,14 @@ else
   # Source the default ci values
   source ./ci/official/envs/ci_default
 
-  # Sourcing TFCI twice, the first time with "-u" unset, means that variable
-  # order does not matter. i.e. "TFCI_BAR=$TFCI_FOO; TFCI_FOO=true" will work.
-  # TFCI_FOO is only valid the second time through.
+  # TODO(angerson) write this documentation
+  # Sources every env, in order, from the comma-separated list "TFCI"
+  # Assumes variables will resolve themselves correctly.
```

Comment:
It might be also good to add to the todo that we reorder them if they are not valid.  I think python needs to come before the platform right now so the py version is defined.  This is a very small nit and better as a follow up from maybe someone else.